pkg/controller/controller.go

```diff
@@ -50,6 +50,18 @@ const (
 	nadPodAnnotation = "k8s.v1.cni.cncf.io/networks"
 )
 
+//Action .... type of action on metrics
+type Action int
```

Comment:
is the 'Action' used only in the package? If so, how about to lower-case (as private scope inside package)

Reply:
fixed